aws/request_test.go

```diff
@@ -572,7 +588,9 @@ func TestRequest_NoBody(t *testing.T) {
 
 			cfg := unit.Config()
 			cfg.Region = "mock-region"
-			cfg.Retryer = aws.DefaultRetryer{NumMaxRetries: 0}
+			cfg.Retryer = aws.NewDefaultRetryer(func(d *aws.DefaultRetryer) {
+				d.NumMaxRetries = 0
```

Comment:
It would be nice to have a no op retryer that always disabled retries, e.g `NopRetryer`